core/templates/dev/head/pages/exploration-editor-page/services/exploration-states.service.ts

```diff
@@ -382,6 +401,12 @@ oppia.factory('ExplorationStatesService', [
         saveStateProperty(
           stateName, 'written_translations', newWrittenTranslations);
       },
+      saveImage: function(stateName, image) {
```

Comment:
I'm not sure how this method is integrated, can you give a brief info on when and how it's get called?